### PR TITLE
feat(SCT-1348): Added description to deleted case notes in timeline

### PR DESCRIPTION
**Pull request by mapleleafjack**
[Link to ticket ](https://hackney.atlassian.net/browse/SCT-1348)

**What**  

- Added extra information to a deleted case notes in the timeline
- Added tests
- Enabled some Case Note deletion test that were disabled by mistake

**Why**  
We need to display the deletion extra information in the timeline (requested by / date / reason for deletion)

image

### CODE REVIEW

File: components/NewPersonView/Event.tsx [hunk 1]
```diff
@@ -48,6 +48,15 @@ const Event = ({ event }: Props): React.ReactElement => {
         {safelyFormat(displayDate)}
         {event.caseFormUrl?.includes('google') && ` · Google document`}
         {event.officerEmail && ` · ${event.officerEmail}`}
+        {event.deleted && event.deletedFields
```
Comment (JohnFarrellDev) [inline components/NewPersonView/Event.tsx:51]: Why do we have the same conditional check 3 times?

Could we just do 

```

{event.deleted && event.deletedFields && (
    ` · deleted ${safelyFormat(event.deletedFields?.dateOfDeletion)}
      · ${event.deletedFields?.deleteReason}
      · requested by ${event.deletedFields?.deleteRequestedBy}
    `
)}

```

File: factories/cases.ts [hunk 2]
```diff
@@ -57,6 +57,12 @@ export const mockedDeletedCaseNote = caseFactory.build({
   },
   formName: 'adult-case-note',
   deleted: true,
+  deletedFields: {
+    deletedBy: 'jack.musajo@hackney.gov.uk',
+    dateOfDeletion: '2020-12-31',
```
Comment (JohnFarrellDev) [inline factories/cases.ts:62]: Are we definitely returning in this format and not an ISO datetime?

Probably fine if we are but just checking

Comment (rachel-hutchinson) [reply factories/cases.ts]: I checked and we are expecting ISO format so I've updated this

**Review by JohnFarrellDev — APPROVED** [review]
LGTM :+1:

**Comment by rachel-hutchinson** [comment]
Holding off merging this into main until the backend is ready

**Comment by JohnFarrellDev** [comment]
> Holding off merging this into main until the backend is ready

could be worth making this a draft PR for now

**Comment by rachel-hutchinson** [comment]
> > Holding off merging this into main until the backend is ready
> 
> could be worth making this a draft PR for now

I took another look and realised that the backend was working as expected but the frontend was just had inconsistent naming. I also added in e2e tests to confirm it all works as expected

**Pull request merged**